rclpy_examples/listener_py.py

```diff
@@ -14,7 +14,13 @@
 
 import sys
 
-import rclpy
+# TODO(jacquelinekay): Remove try block when rclpy supports multiple vendors!!
+try:
+    import rclpy
+except ImportError:
+    print("rclpy was not found. This could be because no valid typesupport was found.")
+    exit()
```

Comment:
It doesn't really matter, especially since it is temporary, but `sys.exit` is probably better to use here: http://stackoverflow.com/a/6501134/671658